kubeflow/trainer/rhai/traininghub_test.py

```
@@ -690,5 +690,127 @@ def test_instrumentation_wrapper_flush_all_prints():
+        ckpt_output_dir=str(ckpt_dir),
+        metrics_port=28080,
+    )
+
+    # Execute the instrumentation code
+    exec_globals = {}
+    exec(wrapper, exec_globals)
+
+    # Should not raise exception when no files exist
+    try:
+        exec_globals["apply_progression_tracking"]()
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)